make_queue/migrations/0016_auto_20201107_1152.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.0.10 on 2020-11-07 10:52
+
+from django.db import migrations, models
+import django.db.models.deletion
+import web.multilingual.database
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('make_queue', '0015_machinetype'),
+    ]
+
+    operations = [
```

Comment:
All of these changes seem to be from the MachineTypeField. Is this something that is missing from the previous migration? @ddabble

Reply:
Yes, that's correct; I forgot to update the migration after having done quite a few rebases 😅 It was added in #330, however, so this migration should be removed after merging with `dev`.